src/cf-typography/src/atoms/links.less

```diff
@@ -126,16 +54,17 @@
             left: 0;
         }
```

Comment:
Yeah, if you take the recommendations in all of my comments, it works. I tested it out in the DevTools.